src/components/Notification/Notification.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Transition } from 'react-transition-group'
+
+import { Button, Icon } from '../../../src'
+
+const propTypes = {
+  item: {
+    show: PropTypes.bool,
+    key: PropTypes.string,
+    type: PropTypes.string,
+    position: PropTypes.string,
+    isDismissable: PropTypes.bool,
+    isWithIcon: PropTypes.bool,
```

Comment:
I would propose to drop the `isWithIcon` prop in favor of only the `icon` one: as for now both have to be in tandem to be used properly, so `icon` is sufficient here.